packages/schema-blocks/src/functions/blocks/parse.ts

```diff
@@ -33,6 +33,7 @@ function parseText( text: string, { separator, instructions }: BlockDefinition )
  * @returns The nodes parsed as leaves.
  */
 function parseNodes( nodes: NodeListOf<ChildNode>, definition: BlockDefinition ): BlockLeaf[] {
+	// eslint-disable-next-line no-use-before-define
```

Comment:
I think this one is covered by `"no-use-before-define": [ "error", { "functions": false } ]`, right?
(To test this, I removed this comment and still had 0 `eslint` warnings)